tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -160,23 +160,52 @@ install_cc_image() {
 	export AA_KBC="${1:-offline_fs_kbc}"
 	image_type="${2:-image}"
 	image_initrd_suffix="${3:-""}"
+	tee="${4:-""}"
 	export KATA_BUILD_CC=yes
 
+	local jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-cc-$(uname -m)/${cached_artifacts_path}"
+	local component="rootfs-image"
+	if [ -n "${tee}" ]; then
+		jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-${tee}-cc-$(uname -m)/${cached_artifacts_path}"
+		component="${tee}-rootfs-image"
+	fi
+
+	local osbuilder_last_commit="$(get_last_modification "${repo_root_dir}/tools/osbuilder")"
```

Comment:
@fidencio don't we need to remove -dirty here too?

Reply:
Not really.  The tree becomes dirty as the result of the rootfs build, that's the reason we remove it in the script we cache the content.

Here, OTOH, we (ideally) won't do any build, that's the reason the `-dirty` doesn't even show up here.

Reply:
cool, I was suspecting that but preferred to ask :)